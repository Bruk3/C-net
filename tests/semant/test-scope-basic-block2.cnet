int main()
{
    {
        int x = 5;
        int y = 10;
        {
            {
                int y = 20;

                x+=1;

                y+=1;
            }
            int z = x + y; // outer block x and y
        }
    }
    return 0;
}