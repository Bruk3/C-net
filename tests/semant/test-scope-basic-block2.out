int main()
{
{
{
int x = (int : 0);
(int : x=(int : 5));
}
{
int y = (int : 0);
(int : y=(int : 10));
}
{
{
{
int y = (int : 0);
(int : y=(int : 20));
}
(int : x=(int : (int : x) + (int : 1)));
(int : y=(int : (int : y) + (int : 1)));
}
{
int z = (int : 0);
(int : z=(int : (int : x) + (int : y)));
}
}
}
return (int : 0);
}

